src/table/TableWrapper.jsx

```diff
@@ -30,6 +30,7 @@ export default function TableWrapper(props) {
   const [tableWidth, setTableWidth] = useState();
   const [page, setPage] = useState(0);
   const [rowsPerPage, setRowsPerPage] = useState(100);
+  const focusedCellCoord = useRef([0, 0]);
```

Comment:
use useRef to set mutable data that is tied to the component instance, but doesn't trigger any renders on change

Reply:
NIce, that is what we want.